Loadouts/msv_ak74_emr.hpp

```diff
@@ -0,0 +1,323 @@
+  backpack[] = {"rhs_assault_umbts"};
+  backpackItems[] = {BASE_MEDICAL};
+  weapons[] = {RIFLE};
+  magazines[] = {RIFLE_MAG,BASE_GRENADES};
+  items[] = {BASE_TOOLS,"rhs_1PN138"}; // rhs_1PN138 russian single tube NVG
+  linkedItems[] = {BASE_LINKED};
+  attachments[] = {RIFLE_ATTACHMENTS};
+  opticChoices[] = {"rhs_acc_pkas","rhs_acc_pgo7v","rhs_acc_1p29","rhs_acc_1p63","rhs_acc_1p78","rhs_acc_1pn93_2","rhs_acc_ekp1"};
+};
+class Fic_Soldier_Carbine: potato_msv_rifleman {// carbine-man
+  weapons[] = {CARBINE};
+  magazines[] = {CARBINE_MAG,BASE_GRENADES};
+};
+class potato_msv_sr: potato_msv_rifleman {// FTL
+  vest[] = {"rhs_6b23_digi_6sh92_vog_headset"};
+  weapons[] = {GLRIFLE};
+  magazines[] = {GLRIFLE_MAG,GLRIFLE_MAG_HE,GLRIFLE_MAG_SMOKE,LEADER_GRENADES};
+  items[] += {LEADER_TOOLS};
+  linkedItems[] += {LEADER_LINKED,BINOS};
+};
+class potato_msv_sl: potato_msv_sr { // SL
+  headgear[] = {"rhs_tsh4"};
+  backpack[] = {"B_FieldPack_oli"};
+  handguns[] = {PISTOL};
+  magazines[] += {PISTOL_MAG};
+  linkedItems[] = {BASE_LINKED,LEADER_LINKED,RANGE_FINDER};
+  items[] += {RADIO_MR,"rhs_6b27m_digi"};
+};
+class potato_msv_plt: potato_msv_sl { // PLT
+  headgear[] = {"rhs_fieldcap_digi"};
+  items[] += {RADIO_LR};
+};
+class potato_msv_aplt: potato_msv_plt {}; // PLT Assistant
+class potato_msv_coy: potato_msv_plt { // COY
+  headgear[] = {"rhs_tsh4"};
+};
+class potato_msv_acoy: potato_msv_plt{}; // COY Assistant
+class potato_msv_pol: potato_msv_rifleman { // Poloitical Officer
+  vest[] = {"rhs_vest_commander"};
+  headgear[] = {"rhs_beret_mp1"};
+  backpack[] = {}; // This loadout looks stupid with a backpack
+  backpackItems[] = {BASE_MEDICAL};
+  weapons[] = {"rhs_weap_makarov_pm"};
+  magazines[] = {"rhs_mag_9x18_8_57N181S:3"};
+  items[] += {RADIO_MR,RADIO_MR}; // 2 radios
+};
+class potato_msv_AR: potato_msv_rifleman {// AR
+  weapons[] = {AR};
+  magazines[] = {AR_MAG,PISTOL_MAG,BASE_GRENADES};
+  handguns[] = {PISTOL};
+  attachments[] = {};
+};
+class potato_msv_g: Fic_Soldier_Carbine {// Grenadier
+  magazines[] += {AT_MAG};
+  launchers[] = {AT};
+};
+class potato_msv_ag: Fic_Soldier_Carbine {// Assistant Grenadier
+  magazines[] += {AT_MAG};
+};
+class potato_msv_sm: Fic_Soldier_Carbine {// Medic
+  vest[] = {"rhs_6b23_digi_medic"};
+  magazines[] = {CARBINE_MAG,MEDIC_GRENADES};
+  backpackItems[] = {MEDIC_MEDICAL};
+};
+class potato_msv_marksman: potato_msv_rifleman {// Squad Marksman
+  weapons[] = {MARKSMAN};
+  magazines[] = {MARKSMAN_MAG};
+  items[] += {"ACE_RangeCard"};
+  attachments[] = {"rhs_acc_pso1m2"};
+};
+class Fic_Spotter: potato_msv_rifleman {
+  linkedItems[] += {RANGE_FINDER};
+};
+class potato_msv_mmgg: potato_msv_AR {// MMG
+  backpack[] = {"B_Kitbag_sgg"};
+  weapons[] = {MMG};
+  magazines[] = {MMG_MAG,PISTOL_MAG,BASE_GRENADES};
+  attachments[] = {};
```

Comment:
I was talking about both, the MMG PKM shouldn't have an optic, the AR RPK could, but it's really not a great one.